share/lib/python/neuron/rxd/species.py

```diff
@@ -1983,7 +1983,7 @@ def __getitem__(self, r):
                         self, self._extracellular_instances[r]
                     )
                 return self._species_on_region[r]
-        raise RxDException("no such region")
+        raise RxDException(f"{repr(self)} is not defined on region {repr{r})")
```

Comment:
Should probably be:

```suggestion
        raise RxDException(f"{repr(self)} is not defined on region {repr(r)}")
```

Looks good otherwise. Would it be possible to add a test for this PR? I see that the lines changed aren't covered by the current ones (see for instance https://app.codecov.io/gh/neuronsimulator/nrn/commit/56650c58f1b63bae729c6d841b6fd248bc5365b6/blob/share/lib/python/neuron/rxd/species.py?dropdown=coverage#L2476)